source/com.microsoft.tfs.client.eclipse.help/content/InstallTEE.htm

```diff
@@ -470,7 +470,7 @@
 						<b>Integrated development environment (IDE) requirements:</b>
 					</p>
 					<ul xmlns="">
-						<li>IDEs that are based on Eclipse 3.5 to 4.4 on Windows, Linux, Mac OS X, Solaris, AIX, or HP-UX.</li>
+						<li>IDEs that are based on Eclipse 3.5 to 4.5 on Windows, Linux, Mac OS X, Solaris, AIX, or HP-UX.</li>
```

Comment:
Is this the place to put the warning:
This is the last version of TEE that will support Eclipse 3.5. Future versions will only support 4.? and higher.
?